Formula/m/mysql.rb

```diff
@@ -45,14 +47,14 @@ class Mysql < Formula
   conflicts_with "mariadb", "percona-server",
     because: "mysql, mariadb, and percona install the same binaries"
 
-  fails_with gcc: "5" # for C++17
+  fails_with :clang do
+    build 1400
+    cause "Requires C++20"
+  end
 
-  # Patch out check for Homebrew `boost`.
-  # This should not be necessary when building inside `brew`.
-  # https://github.com/Homebrew/homebrew-test-bot/pull/820
-  patch do
-    url "https://raw.githubusercontent.com/Homebrew/formula-patches/bd61f2edc4c551856f894d307140b855edb2b4f5/mysql/boost-check.patch"
-    sha256 "b90c6f78fa347cec6388d2419ee4bc9a5dc9261771eff2800d99610e1c449244"
```

Comment:
Probably should've just updated this

Reply:
Yeah, I realized that when working on ICU PR. Just re-added patch there. I think it usually only happens when we do mega ICU PRs.